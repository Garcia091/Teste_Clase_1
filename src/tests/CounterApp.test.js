import React from 'react'
import {shallow} from 'enzyme'
import CounterApp from '../CounterApp'
import '@testing-library/jest-dom'

describe('pruebas <CounterApp/>', () => {

    test('Validar render de <PrimeraApp/>', () => {
        const component = shallow(<CounterApp/>)
        expect(component).toMatchSnapshot(); 
    })   
    
    test('Validar el valor inicial ', () => {
        
        const component = shallow(<CounterApp value ={100} />)

        const valueCount = component.find('h2').text().trim();

        expect(valueCount).toBe('100')
        
    })
    

})