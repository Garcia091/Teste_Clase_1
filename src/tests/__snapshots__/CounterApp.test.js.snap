// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pruebas <CounterApp/> Validar render de <PrimeraApp/> 1`] = `
<Fragment>
  <h1>
    Contador
  </h1>
  <h2>
     
    10
     
  </h2>
  <button
    onClick={[Function]}
  >
    +1
  </button>
  <button
    onClick={[Function]}
  >
    Reset
  </button>
  <button
    onClick={[Function]}
  >
    -1
  </button>
</Fragment>
`;
